perma_web/perma/models.py

```diff
@@ -1997,14 +1996,23 @@ def write_uploaded_file(self, uploaded_file, cache_break=False):
                           user_upload='True',
                           content_type=mime_type,
                           url=warc_url)
```

Comment:
We'll want to make another `Capture` object here, with `role='provenance_summary'`, to go along with the primary.

See also the discussion about whether, at this time, we should remove the random string appended to the end. If we are going to do that, I think we should do it now rather than later, make the corpus more consistent.

Reply:
> the discussion about whether, at this time, we should remove the random string appended to the end

I haven't seen this discussion yet -- is it a comment on this PR, or elsewhere?

Reply:
It is probably in the Liner ticket for studying and designing this. I'll try and Slack you a link.